backend/src/main/resources/db/migration/V23__Add_match_status.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE matches ADD COLUMN status SMALLINT NOT NULL DEFAULT 0;
+ALTER TABLE matches ALTER status DROP DEFAULT;
```

Comment:
What's the point of dropping the default right after you set it? Just remove the default from the first line no?

Reply:
If we don't, then we can't add the column due to the not null constraint. It needs a default value to fill in the rows already there.

Reply:
Wait why are we having a not null with no default? That doesn't sound good

Reply:
Not null with no default is pretty standard. It just means everything you insert has to have that column populated.

Reply:
Yeah that makes sense now that you explained it thanks